engine_modules/market/models.py

```diff
@@ -29,7 +29,32 @@ class Meta:
 	# whereas the one with turn n-1 has the values at beginning of turn.
 	# This is why we have default turn 0 and we need to initialize both for turn 0 and 1
 	turn = models.PositiveSmallIntegerField(default=0)
+	# The meaning of 'value' and 'bubble_value' changed: value is now the market assets + the bubble modifier.
+	# bubble_value is only here as a token for the bubble: 0 -> no bubble, 1 -> domination, -1 -> 'dry' bubble
+	# This should greatly simplify the DB requests for the CorporationMarkets that have a bubble
 	value = models.SmallIntegerField()
+	# bubble_value should only be modified through the update_bubble() method
+	bubble_value = models.SmallIntegerField(default=0)
```

Comment:
since values are heavily constrained and not aritmetical one, more flags, maybe using a `choices` would be better?

bubble_value=NONE | DOMINATION | DRY